packages/fast-tooling-react/src/data-utilities/mapping.ts

```diff
@@ -32,20 +32,26 @@ export function mapDataToComponent(
     plugins: Array<Plugin<PluginProps>> = []
 ): MappedDataLocation[] {
     const mappedData: any = cloneDeep(data);
+    let reactChildrenDataLocations: string[] = [];
+    let pluginModifiedDataLocations: PluginLocation[] = [];
+
+    if (childOptions.length > 0) {
```

Comment:
Do we want to check that `length` exists first? (line 47 as well)

Reply:
You can see the value initializes with an empty array, it is never undefined.